lib/perl/Genome/Sys/LSF/ResourceParser.pm

```diff
@@ -79,6 +81,42 @@ sub _create_option_spec {
     return ("$option=s" => sub { my ($option_name, $value, )});
 }
 
+sub null_formatter {
+    my $value = shift;
+    return $value;
```

Comment:
Is it ever important that the value definitely be a string and not a number (i.e. `return "$value";`)?